src/System.Memory/src/System/Buffers/BuffersExtensions.cs

```diff
@@ -103,6 +107,7 @@ public static T[] ToArray<T>(in this ReadOnlySequence<T> sequence)
         /// <summary>
         /// Writes contents of <paramref name="value"/> to <paramref name="writer"/>
         /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static void Write<T>(this IBufferWriter<T> writer, ReadOnlySpan<T> value)
```

Comment:
This is an interface call..

Reply:
Its a non-virtual static call? A static method can't implement an interface

Reply:
And hopefully when its inlined and [called with](https://github.com/aspnet/SignalR/blob/9fd713c73a1fadf49ea00bc500c0d456281aede2/src/Common/MemoryBufferWriter.cs#L254-L268) the `sealed class` `MemoryBufferWriter` the Jit will devirtalize it as the interface call inside cannot be any other type? /cc @AndyAyersMS